libraries/lib-ffmpeg-support/FFmpegTypes.h

```diff
@@ -103,8 +103,8 @@ using AVDiscardFwd = int;
 // Stable structures and emums
 
 struct AudacityAVRational {
-    int num; ///< numerator
-    int den; ///< denominator
+    int num = 0; ///< numerator
+    int den = 1; ///< denominator
```

Comment:
I strongly disagree with this change. FFmpeg behavior may differ if `den=0`. For example, it means that the decoder is responsible for setting up time bases based on stream content. But I agree that it is probably better to initialize the fields explicitly, regardless that this is a DTO for the C structure,

Reply:
Well defined values should be there I think if this struct is ever default constructed.  You decide which values.